src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoCurrencyDecimalDigits.cs

```diff
@@ -11,80 +12,38 @@ namespace System.Globalization.Tests
 {
     public class NumberFormatInfoCurrencyDecimalDigits
     {
-        // PosTest1: Verify property CurrencyDecimalDigits default value
-        [Fact]
-        public void PosTest1()
-        {
-            NumberFormatInfo nfi = new NumberFormatInfo();
-            Assert.Equal(2, nfi.CurrencyDecimalDigits);
-        }
-
-        // PosTest2: Verify set value of property CurrencyDecimalDigits
-        [Fact]
-        public void PosTest2()
-        {
-            NumberFormatInfo nfi = new NumberFormatInfo();
-
-            for (int i = 0; i < 100; i++)
-            {
-                nfi.CurrencyDecimalDigits = i;
-                Assert.Equal(i, nfi.CurrencyDecimalDigits);
-            }
-        }
-
-        // TestArgumentOutOfRange: ArgumentOutOfRangeException is thrown
-        [Fact]
-        public void TestArgumentOutOfRange()
-        {
-            VerificationHelper<ArgumentOutOfRangeException>(-1);
-            VerificationHelper<ArgumentOutOfRangeException>(100);
-        }
-
-        // TestInvalidOperation: InvalidOperationException is thrown
-        [Fact]
-        public void TestInvalidOperation()
+        public static IEnumerable<object[]> CurrencyDecimalDigits_TestData()
         {
-            NumberFormatInfo nfi = new NumberFormatInfo();
-            NumberFormatInfo nfiReadOnly = NumberFormatInfo.ReadOnly(nfi);
-            Assert.Throws<InvalidOperationException>(() =>
-            {
-                nfiReadOnly.CurrencyDecimalDigits = 1;
-            });
+            yield return new object[] { NumberFormatInfo.InvariantInfo, 2, 2 };
+            yield return new object[] { new CultureInfo("en-US").NumberFormat, 2, 2 };
+            yield return new object[] { new CultureInfo("ko").NumberFormat, 0, 2 }; // Todo: determine why some values are different
```

Comment:
The TODO can be removed. ICU had some values that didn't seem right at the time but after spending more time looking at other inconsistencies between windows and ICU the results are expected and don't require further research.